pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -947,18 +945,33 @@ func buildDefaultCluster(env *model.Environment, name string, discoveryType apiv
 		}
 	}
 
-	defaultTrafficPolicy := buildDefaultTrafficPolicy(env, discoveryType)
+	defaultTrafficPolicy := buildDefaultTrafficPolicy(env, discoveryType, direction, port)
 	applyTrafficPolicy(env, cluster, defaultTrafficPolicy, nil, nil, "",
 		DefaultClusterMode, direction)
 	return cluster
 }
 
-func buildDefaultTrafficPolicy(env *model.Environment, discoveryType apiv2.Cluster_DiscoveryType) *networking.TrafficPolicy {
+func buildDefaultTrafficPolicy(env *model.Environment, discoveryType apiv2.Cluster_DiscoveryType,
+	direction model.TrafficDirection, port *model.Port) *networking.TrafficPolicy {
 	lbPolicy := DefaultLbType
 	if discoveryType == apiv2.Cluster_ORIGINAL_DST {
 		lbPolicy = networking.LoadBalancerSettings_PASSTHROUGH
 	}
+
+	// Set default circuit breaking for Envoy graceful shutdown
+	// Values derived from Envoy docs https://www.envoyproxy.io/docs/envoy/latest/faq/transient_failures#outlier-detection
+	var outlierDetectionPolicy *networking.OutlierDetection
```

Comment:
Same as above - at this point in 1.1 ( and in general ) any change that affects the behavior should have a way to turn off, env variables in pkg/../features/pilot is one way. 

After they get tested we can remove the feature flag, but it is impossible to know how this will impact all users.